src/components/MovieTile.jsx

```diff
@@ -20,9 +21,10 @@ const useOutsideClick = (menuRef, setMenuTileContextOpen) => {
     }, [menuRef, setMenuTileContextOpen]);
 }
 
-const MovieTile = ({movieData, onClick}) => {
+const MovieTile = ({movieData}) => {
```

Comment:
add type for  movieData

Reply:
Module 10 has task to move all code to typescript